tests/net/mld/src/main.c

```diff
@@ -330,42 +331,58 @@ static void send_query(struct net_if *iface)
 	zassert_not_null(pkt, "Cannot allocate pkt");
 
 	net_pkt_set_ipv6_hop_limit(pkt, 1); /* RFC 3810 ch 7.4 */
-	net_ipv6_create(pkt, &peer_addr, &dst);
+	ret = net_ipv6_create(pkt, &peer_addr, &dst);
+	zassert_true(!ret, "Cannot create ipv6 pkt");
```

Comment:
I would be more clarer if you write this as `zassert_false(ret, ...);` Please update all invocations.

Reply:
Thanks for comment.
I updated the patch.